routes/api.js

```diff
@@ -72,7 +73,7 @@ router.use(function(req, res, next) {
  * @param  {Object} (req, res)                                Anonymous function to handle request and response
  */
 router.get('/getUserInfo', authenticationHelpers.isAuthOrRedirect, (req, res) => {
-    res.status(200).json({user: req.user});
+    res.status(200).json({user: req.user.email});
```

Comment:
Why was this change made? This might affect the file like navbar.component.ts which consumes this API.

Reply:
Actually, this API  was not used anywhere in the application until now. In navbar component, we use **get Username** API. 
From this API I get the user's email which I use in Leaderboard component to highlight the current user.